use-cases/auditbeat.yml

```diff
@@ -73,75 +73,74 @@ fields:
     type: date
     description: The last change time of the file (time when metadata was changed).
 
-# TODO (@ruflin 2018-05-01): These fields are not in ECS. Needs decision or removal.
-#
-#- name: hash
-#  group: 3
-#  description: >
-#    Hash fields used in Auditbeat.
-#
-#    The hash field contains cryptographic hashes of data associated with the event
-#    (such as a file). The keys are names of cryptographic algorithms. The values
-#    are encoded as hexidecimal (lower-case).
-#
-#    All fields in user can have one or multiple entries.
-#  fields:
-#    - name: md5
-#      type: keyword
-#      description: >
-#        MD5 hash.
-#
-#    - name: sha1
-#      type: keyword
-#      description: >
-#        SHA-1 hash.
-#
-#    - name: sha224
-#      type: keyword
-#      description: >
-#        SHA-224 hash (SHA-2 family).
-#
-#    - name: sha256
-#      type: keyword
-#      description: >
-#        SHA-256 hash (SHA-2 family).
-#
-#    - name: sha384
-#      type: keyword
-#      description: >
-#        SHA-384 hash (SHA-2 family).
-#
-#    - name: sha512
-#      type: keyword
-#      description: >
-#        SHA-512 hash (SHA-2 family).
-#
-#    - name: sha512_224
-#      type: keyword
-#      description: >
-#        SHA-512/224 hash (SHA-2 family).
-#
-#    - name: sha512_256
-#      type: keyword
-#      description: >
-#        SHA-512/256 hash (SHA-2 family).
-#
-#    - name: sha3_224
-#      type: keyword
-#      description: >
-#        SHA3-224 hash (SHA-3 family).
-#
-#    - name: sha3_256
-#      type: keyword
-#      description: >
-#        SHA3-256 hash (SHA-3 family).
-#
-#    - name: sha3_384
-#      type: keyword
-#      description: >
-#        SHA3-384 hash (SHA-3 family).
-#
-#    - name: sha3_512
-#      type: keyword
-#      description: >
-#        SHA3-512 hash (SHA-3 family).
+
+- name: hash
```

Comment:
There are at least 2 more hash types in Auditbeat now - blake2b and xxhash. Would be good to add them so that people use the same names.

https://github.com/elastic/beats/blob/master/auditbeat/module/file_integrity/_meta/fields.yml#L12-L22

https://github.com/elastic/beats/blob/master/auditbeat/module/file_integrity/_meta/fields.yml#L72-L74